tests/class/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -317,8 +319,8 @@ abstract class ReallyReallyReallyLongClassName
     {
         parent::__construct($test_int ?: 1);
         $this->other = $test_string;
-        $this->current_version =
-            $current_version ?: new Content_Version_Model();
+        $this->current_version = $current_version
+            ?: new Content_Version_Model();
```

Comment:
Can we add test when in it on one line?

Reply:
I think we already have a test where its on 1 line? https://github.com/prettier/plugin-php/blob/master/tests/retif/__snapshots__/jsfmt.spec.js.snap#L29